src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for printing personal information as string.
```

Comment:
This interface need not be specific to address book at all. So references to addresbook or its components can be avoided.